packages/xstate-vue/test/UseService.vue

```diff
@@ -17,8 +17,7 @@ export default defineComponent({
     }
   },
   setup(props) {
-    const { service } = toRefs(props);
```

Comment:
was this needed? I've figured out that maybe not and using it here has been creating a type problem